pkg/micropython/Makefile.dep

```diff
@@ -0,0 +1,8 @@
+USEMODULE += xtimer
+USEMODULE += stdin
+
+# MicroPython doesn't compile for <32bit platforms
+FEATURES_BLACKLIST += arch_8bit arch_16bit
```

Comment:
```suggestion
FEATURES_REQUIRED += arch_32bit
```

Reply:
that will require extra work when we support native 64bit, or fixup 16bit (which IIRC upstream should be able to do). Explicit is good!